custom-vu/src/main/kotlin/jces1209/vu/memory/BoardPagesMemory.kt

```diff
@@ -0,0 +1,26 @@
+package jces1209.vu.memory
+
+import com.atlassian.performance.tools.jiraactions.api.SeededRandom
+import jces1209.vu.page.boards.browse.BoardList
+import jces1209.vu.page.boards.view.BoardPage
+
+class BoardPagesMemory(
+    val seededRandom: SeededRandom
+) {
+    val kanban = SeededMemory<BoardPage>(seededRandom)
+    val scrum = SeededMemory<BoardPage>(seededRandom)
+    val nextGen = SeededMemory<BoardPage>(seededRandom)
+
+    fun rememberBoardsList(boardsList: BoardList.MixedBoards) {
+        kanban.remember(boardsList.kanban)
+        scrum.remember(boardsList.scrum)
+        nextGen.remember(boardsList.nextGen)
+    }
+
+    fun getMap() : Map<String, SeededMemory<BoardPage>>{

```

Comment:
The `String` is only used to generate `ActionMeter.measure(key)`s. We can add `val typeLabel: String` to `BoardPage` and remove the `getMap` and `BoardList.boardName*`.

- [ ] consider including attributes into the abstraction instead of tracking them externally